lib/Frontend/Frontend.cpp

```diff
@@ -421,7 +427,8 @@ shouldImplicityImportSwiftOnoneSupportModule(CompilerInvocation &Invocation) {
     return false;
 
   if (shouldImportSwiftOnoneModuleIfNoneOrImplicitOptimization(
-          Invocation.getFrontendOptions().RequestedAction)) {
+          Invocation.getFrontendOptions().RequestedAction,
+          Invocation.getFrontendOptions().TrackSystemDeps)) {
     return true;
   }
   return Invocation.getFrontendOptions().isCreatingSIL();
```

Comment:
This bit here's still going to trip you up, yeah? Actually, isn't this basically the same as what `shouldImportSwiftOnoneModuleIfNoneOrImplicitOptimization` is doing?

Reply:
Oh, right.  Meant to collapse these!